tools/verdict-back-ends/verdict-bundle-parent/verdict-bundle/pom.xml

```diff
@@ -23,6 +23,11 @@
             <groupId>com.ge.research.osate.verdict</groupId>
             <artifactId>com.ge.research.osate.verdict.vdm</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.ge.verdict</groupId>
+            <artifactId>verdict-assurance-case</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
We would have to change this place too if we ever change VERDICT's version number.  There should be only one place in each pom that has VERDICT's version number (the `<parent>` section).  We use `${project.version}` instead of 1.0.0-SNAPSHOT everywhere else.

Furthermore, this dependency shouldn't even have a `<version>` at all.  We normally set all dependency versions inside `<dependencyManagement>` in tools/pom.xml so there is only one place to change dependency versions too.  Please add this verdict-assurance-case dependency to tools/pom.xml (with `${project.version}`) and remove the `<version>` line here.

Reply:
Done